src/pages/workspaces/workspace/Data.js

```diff
@@ -226,23 +227,21 @@ class WorkspaceData extends Component {
        * TODO: once column selection is implemented, add another hidden input with name: 'attributeNames' and
        * value: comma-separated list of attribute names to support downloading only the selected columns
        */
-      tooltip({
-        component: buttonPrimary({ type: 'submit', disabled: !orchestrationRoot }, [
+      h(TooltipTrigger, { content: 'Download all data as a file' }, [
```

Comment:
You mean centered with respect to the tooltip body, not the point? The current design has tooltips jutting to the side, so this would require a rethink of that. Let's discuss later.